examples/visualization/plot_evoked_arrowmap.py

```diff
@@ -0,0 +1,50 @@
+"""
+=============================================
+Plotting topographic arrowmaps of evoked data
+=============================================
+
+Load evoked data and plot arrowmaps along with the topomap for selected time
+points. Arrowmap is based upon Hosaka-Cohen transformation and represents
+actual current underneath the MEG sensors, they are poor man MNE
```

Comment:
Typos:

> Arrowmap is based upon Hosaka-Cohen transformation and represents
actual current underneath the MEG sensors, they are poor man MNE

should be:

An arrowmap is based upon the Hosaka-Cohen transformation and represents
the actual current underneath the MEG sensors, they are a poor man's MNE.

But then, it cannot represent the **actual current** can it? Indeterminacy of the inverse and all that. May I suggest:

An arrowmap is based upon the Hosaka-Cohen transformation and represents
an estimation of the current flow underneath the MEG sensors, they are a poor man's MNE.

Reply:
Actually I copied David's actual word, but definitely estimation is the right word.  Will update.